### ISSUE TITLE
Grand Theft Auto's DOS version crashes

### ISSUE THREAD

**Issue by atylmo** [86Box/86Box#1289]
**Describe the bug**
The Windows 9x version works, but the DOS version will spit out an error in the prompt that I can't quite catch unless I run it in pure DOS mode. It is similar to [this](http://www.3000ad.com/forum/topic/1000532-my-game-dont-want-to-start-any-more/) error, which I cribbed from to type mine out because I didn't have a convenient way of copying it handy.

I also get a `Crash address (unrelocated) = 1:00102F37` message at the end but that seems to only show up when launched in Windows.

```
DOS/4GW Professional error (2001): exception 0Dh (general protection fault) at 2
D8:1018C0D
TSF32: prev_tsf32 7B44
SS D0 DS 20 ES 60 FS 0 GS 0
EAX 100 EBX 0 ECX 0 EDX 100
ESI 10210000 EDI 10 EBP 101B03A2 ESP 7B18
CS:IP 2D8:1018AC0D ID 0D COD 0 FLG 2
CS= 2D8, USE32, page granular, limit FFFFFFFF, base 0, acc CF9B
SS= D0, USE32, byte granular, limit 88BF, base 146E20, acc 4093
DS= 20, USE16, byte granular, limit FFFF, base C3B0, acc 93
ES= 60, USE16, byte granular, limit FFFF, base C3B0, acc 93
FS= 0, USE16, byte granular, limit 0, base 16, acc 0
GS= 0, USE16, byte granular, limit 0, base 16, acc 0
CR0: PC:0 ET:1 TS:0 EM:0 MP:0 PE:1 CR2: 101BC0DB CR3: 16F000
```

**To Reproduce**
1. Start the game either in DOS or in Windows with the shortcut (not the "Windows 95" version)
2. Choose any video mode at the "Run GTA" menu
3. When the "DOS/4GW" screen appears, the game will crash after a few seconds and show the above error message. In Windows, the text in the DOS prompt becomes large and stretched out.

**Expected behavior**
The game should continue normally.

**Desktop (please complete the following information):**
 - OS: Host: Windows 10 x64, Guests: Windows 95 OSR2, Windows 98 SE
 - 86Box version: v3.0 2808
 - Build type: regular

**Additional context**
It works in PCem v15 but v17 also has the issue. I'm not sure which 86Box version would correlate with it.  Looking up the error above it seems to often be an audio driver problem but even if I select "No digital audio" in the game it still crashes out. Conversely, in the working version, it works whether the sound is disabled or enabled.

**Emulated system**
Shuttle-HOT-557
Pentium MMX 200
128 MB RAM
ATI Graphics Pro Turbo (Mach64 GX)
SoundBlaster 16
Voodoo 1 cards in SLI with 8 MB each

**Comment by Diicorp95** [comment]
This may be a bug in the **game itself**.

**Comment by OBattler** [comment]
Which 86Box build exactly introduced this problem?

**Comment by OBattler** [comment]
The game appears to work for me right now, see if it also works for you.

**Comment by OBattler** [comment]
Given that it appears to work for me, I think it's safe to close this - it can be reopened in case someone can still reproduce it.

**Comment by atylmo** [comment]
Hey, sorry. I gave it a try with regular build `b931e814` but my graphics driver got messed up. Had low color depth on the desktop and got an error about `desk.cpl` when I tried to change it. I got a black screen when I launched the game but I got past that and still got the error once the game loaded. I'll try again with a fresh installation of 98.

I haven't tried old 86Box versions but it broke somewhere between PCem v14 and v15.
